tests/mpi/step-40.cc

```diff
@@ -157,14 +157,15 @@ namespace Step40
     DoFTools::make_sparsity_pattern(dof_handler, csp, constraints, false);
     SparsityTools::distribute_sparsity_pattern(
       csp,
-      dof_handler.n_locally_owned_dofs_per_processor(),
+      dof_handler.compute_n_locally_owned_dofs_per_processor(),
```

Comment:
Did you update the tutorial programs `step-18`, `step-40`, `step-55` and `step-62` as well?

Reply:
Not yet... :wink: In any case, I opened issue #8306 because I want to get rid of this argument in the sparsity pattern computation (but I don't use matrices that regularly, so I could forget it without).